<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityEdit</class>
 <widget class="QWidget" name="ActivityEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_activityName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Название активности</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_calories">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Колличество расходуемых калорий</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_activityDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lineEdit_activityName</zorder>
  <zorder>lineEdit_calories</zorder>
  <zorder>textEdit_activityDescription</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>pushButton_cancel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
